package-lock.json

```diff
@@ -9335,9 +9335,9 @@
       "optional": true
     },
     "node_modules/@types/react": {
-      "version": "18.2.9",
```

Comment:
package-lock shouldn't have changed here because you didn't add any new dependencies. I think you need to rebase the branch onto the latest version of dev, making sure to git pull.
Please try to make it so there are no changes to package lock